test/cache-fs.spec.ts

```diff
@@ -1,38 +1,49 @@
 import {EventEmitter} from 'eventemitter3';
+import {InternalEventsEmitter} from '../src/utils';
 import * as Promise from 'bluebird';
 import {expect} from 'chai';
 import {EventsMatcher} from '../test-kit/drivers/events-matcher';
 import {SlowFs} from '../test-kit/drivers/slow-fs';
-import {FileSystem} from '../src/api';
+import {FileSystem, fileSystemEventNames} from '../src/api';
 import {CacheFs} from '../src/cache-fs';
 import {MemoryFileSystem} from '../src/memory-fs';
-import {assertFileSystemContract, ignoredDir, ignoredFile} from './implementation-suite';
+import {
+    assertFileSystemContract,
+    ignoredDir,
+    ignoredFile,
+    fileName,
+    dirName,
+    content
+} from './implementation-suite';
 
 describe(`the cache file system implementation`, () => {
 
+    const eventMatcherOptions: EventsMatcher.Options = {
+        interval: 1,
+        noExtraEventsGrace: 10,
+        timeout: 30
+    };
+
     assertFileSystemContract(
         () => Promise.resolve(new CacheFs(new MemoryFileSystem(undefined, [ignoredDir, ignoredFile]))),
-        {
-            interval: 1,
-            noExtraEventsGrace: 10,
-            timeout: 30
-        }
+        eventMatcherOptions
     );
 
     describe(`using slow FileSystem`, () => {
-        const fileName = 'foo.txt';
-        const content = 'content';
         const timeout = 200;
 
         let timer;
         let fs: FileSystem;
         let slow: FileSystem;
         let startTimestamp: number;
+        let matcher: EventsMatcher;
 
         beforeEach(() => {
             startTimestamp = Date.now();
             slow = new SlowFs(timeout)
             fs = new CacheFs(slow)
```

Comment:
here it is https://github.com/wix/kissfs/pull/56/files#diff-92eb11e65174526ae628ddafea6ae249R116